chapter3.adoc

```diff
@@ -244,3 +244,30 @@ prior to the `MINVAL.SPA`, and
 subsequent to the `MFENCE.SPA`
 
 `MINVAL.SPA` is only valid in M-mode.
+
+=== Extension to behavior of SFENCE.VMA and HFENCE.VMA
+
+The behavior of the SFENCE.VMA and the HFENCE.VMA instructions are affected when
+supervisor domains are used by the M-mode RDSM.
```

Comment:
These fences are affected whenever the `Smsdid` extension is implemented and the mttp.SDID field exists, whether or not M-mode software contains any support for supervisor domains.

```suggestion
the Smsdid extension is implemented.
```